subcommands/targets/offline-update.go

```diff
@@ -396,3 +400,15 @@ func untar(r io.Reader, dstDir string) error {
 
 	return nil
 }
+
+func getBundleTargetsMeta(bundleTufPath string) (bundleTargets *tuf.Signed, err error) {
+	if b, readErr := os.ReadFile(path.Join(bundleTufPath, "bundle-targets.json")); readErr == nil {
+		var foundBundleTargets tuf.Signed
+		if err = canonical.Unmarshal(b, &foundBundleTargets); err == nil {
+			bundleTargets = &foundBundleTargets
+		}
```

Comment:
The `bundleTargets` won't be null if the unmarshal fails.

Reply:
If the unmarshal fails then bundleTargets is supposed to be nil. It won't be so if I apply your suggestion.

Reply:
>In case of an error the bundleTargets should be disregarded i.e. it can be anything.

It should be, but it's not always is so. So, better to return nil.

>This is a normal practice in Golang which allows simplifying the code flow: less spaghetti code, which is already an inherent problem in Golang.

This is very subjective. I wouldn't say that this is a normal practice. It really depends on the agreement and rules agreed on specific project. The golang spec doesn't say anything about it.